py-polars/polars/series/series.py

```diff
@@ -3578,6 +3578,42 @@ def search_sorted(
             If 'any', the index of the first suitable location found is given.
             If 'left', the index of the leftmost suitable location found is given.
             If 'right', return the rightmost suitable location found is given.
+
+        Examples
+        --------
+        >>> s = pl.Series("set", [1, 2, 3, 4, 4, 5, 6, 7])
+        >>> s.search_sorted(4)
+        4
+        >>> s.search_sorted(4, "any")
+        4
```

Comment:
`'any'` is the default, perhaps just show the first one rather than repeating it?

Reply:
Thank you Marco - that makes sense to me.  I removed those lines per your suggestion.